<launch>

<node pkg="forklift_server" type="gui_megapose.py" name="gui_megapose" output="screen">
    <!--Subscriber Topic setting/-->
    <param name="odom_topic" value="/wheel_odom" /><!--里程計 Topic（map -> base_link, 是tf相對關係, 非輪式里程計）/-->
    <param name="shelf_topic" value="/shelf_filter" /><!--車體相機對位AprilTag Topic/-->
    <param name="pallet_topic" value="/pallet_filter" /><!--牙叉相機對位AprilTag Topic/-->
    <param name="forkpos" value="/forklift_pose" /><!--牙叉編碼器回傳Topic/-->
    <param name="offset_x" value="0.0" /><!--對位目標點與tag的左右偏移量/-->

</node>
</launch>
